### DISCUSSION TITLE
新闻内容字体太小

### DISCUSSION THREAD

**Discussion by ghost** [xiaoyaocz/flutter_dmzj#113] [General]
![Screenshot_2023-03-15-21-00-41-005_com xycz dmzjx](https://user-images.githubusercontent.com/17699605/225316588-c5b71185-fe4b-4cbe-a000-0b7063d1c842.jpg)
希望支持调整大小，看截图感觉视频缩略图比例也有问题

**Comment by xiaoyaocz** [comment]
动漫之家新闻Web视频比例就是这样的。
字体大小下版本会增加一个选项。

**Reply by ghost** [reply]
开发方面我不太懂，不知道能不能抓取新闻页面文字图片重新排版调整字体

**Comment by xiaoyaocz** [comment]
https://github.com/xiaoyaocz/flutter_dmzj/releases/tag/v2.0.3-beta
